storage/innobase/buf/buf0dblwr.cc

```diff
@@ -385,8 +386,16 @@ void buf_dblwr_t::recover()
     fil_space_t *space= fil_space_t::get(space_id);
 
     if (!space)
-      /* The tablespace that this page once belonged to does not exist */
+    {
+      /* These pages doesn't belong to existing tablespace.
+      There is a possibility that these pages could be
```

Comment:
We’re processing one page at a time, so I would suggest to write in singular form:
```c
/* This page does not appear to belong to any tablespace.
There is a possibility that this page … */
```